### PR TITLE
New rule to match clients 20 or greater

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: spec/models/rules/age_greater_than_twenty_one_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      it 'matches 1' do
+        expect(clients_that_dont_fit.count).to eq(1)
+      end
+      it 'contains Bob (20 y.o.)' do
+        expect(clients_that_dont_fit.ids).to include bob.id
+      end
+      it 'does not contain Roy (Happy Birthday!)' do
+        expect(clients_that_dont_fit.ids).to_not include roy.id
+      end
+      it 'does not contain Mary (20 y.o)' do
```
Comment (kwiliarty) [inline spec/models/rules/age_greater_than_twenty_one_spec.rb]: Extremely minor: Mary is 22 y.o.

File: spec/models/rules/age_greater_than_twenty_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      it 'matches 1' do
+        expect(clients_that_dont_fit.count).to eq(1)
+      end
+      it 'contains Bob (19 y.o.)' do
+        expect(clients_that_dont_fit.ids).to include bob.id
+      end
+      it 'does not contain Roy (Happy Birthday!)' do
+        expect(clients_that_dont_fit.ids).to_not include roy.id
+      end
+      it 'does not contain Mary (19 y.o)' do
```
Comment (kwiliarty) [inline spec/models/rules/age_greater_than_twenty_spec.rb]: Again very minor, but it would make the test easier to read: Mary is 21 in this test.

**Review by kwiliarty — APPROVED** [review]
I noticed only two very minor issues with the description lines for the tests. The new rules showed up as expected in a browser.

File: spec/models/rules/age_greater_than_twenty_one_spec.rb (hunk 1 above)
Comment (eanders) [reply spec/models/rules/age_greater_than_twenty_one_spec.rb]: Thanks!

File: spec/models/rules/age_greater_than_twenty_spec.rb (hunk 2 above)
Comment (eanders) [reply spec/models/rules/age_greater_than_twenty_spec.rb]: Thanks!

**Pull request merged**